Dockerfile.mediator

```diff
@@ -8,9 +9,10 @@ ENV DJANGO_OSMAXX_CONVERSION_SERVICE_USERNAME=default_user DJANGO_OSMAXX_CONVERS
 ENV NUM_WORKERS=5 DATABASE_HOST=mediatordatabase DATABASE_PORT=5432 APP_PORT=8901 APP_HOST=0.0.0.0
 
 MAINTAINER HSR Geometalab <geometalab@hsr.ch>
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
Should we reset this at the end of the Dockerfile? (I think it'll retain the value even into the running containers, if we don't. That might be unexpected when using apt in such a  container interactively, e.g. for debugging.)

Reply:
Good point.